modules/apps/foundation/roles/roles-admin-web/bnd.bnd

```diff
@@ -1,5 +1,6 @@
 Bundle-Name: Liferay Roles Admin Web
 Bundle-SymbolicName: com.liferay.roles.admin.web
 Bundle-Version: 1.0.10
+Export-Package: com.liferay.roles.admin.web.util
```

Comment:
It was exported so that the test could use it.  I could just move the test into the same module so we don't have to expose this package.